src/style.css

```
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Averin";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Averin";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bigBar {
+  margin: 20px 0;
+}
+
+.input {
+  font-family: "Averin", sans-serif;
```

```suggestion
  font-family: inherit;
```
Apply font-family to the `html` tag selector